elements/bulbs-dfp/bulbs-dfp.js

```diff
@@ -32,7 +32,7 @@ export default class BulbsDfp extends BulbsHTMLElement {
 
     let defaultRefreshInterval = 30000;
     this.adUnitData = this.adsManager.adUnits.units[this.dataset.adUnit];
-    if (!this.adUnitData.refreshDisabled) {
+    if (!this.adUnitData.refreshDisabled && !this.hasAttribute('no-refresh')) {
```

Comment:
should this be opposite? if it has `no-refresh` it should not set a refresh interval?